python/grass/temporal/univar_statistics.py

```diff
@@ -65,12 +65,9 @@ def compute_univar_stats(registered_map_info, stats_module, fs, rast_region=Fals
 
     if not univar_stats:
         gs.warning(
-            _(
-                "Unable to get statistics for {voxel}raster map "
-                "<{rmap}>".format(
-                    rmap=id, voxel="" if stats_module.name == "r.univar" else "3d "
-                )
-            )
+            _("Unable to get statistics for raster map <%s>") % id
```

Comment:
Why did you choose % and %s for formatting? Because of string on line 105?

Reply:
To make existing translations in 4 languages (but now not collected in the templates anymore) match up again. See the long comment analyzing that that you replied to.